components/tools/OmeroPy/src/omero/plugins/sessions.py

```diff
@@ -820,11 +823,19 @@ def run(self):
             t.event.set()
 
     def file(self, args):
+        """Return the file associated to the current active sessions"""
         store = self.store(args)
         srv, usr, uuid, port = store.get_current()
         if srv and usr and uuid:
             self.ctx.out(str(store.dir / srv / usr / uuid))
 
+    def key(self, args):
+        """Return the key associated to the current active sessions"""
+        store = self.store(args)
+        srv, usr, uuid, port = store.get_current()
+        if uuid:
```

Comment:
Can do. In that case, I was mirroring the behavior of `session file` which does not throw a CLI exception is no local session file is found. Either of them is fine.